deploy/olm/csv-rhel/storageos.clusterserviceversion.yaml

```
@@ -394,7 +394,21 @@ spec:
           added to the StorageOS role.
         displayName: Disable Fencing
         path: disableFencing
-      - description: When enabled, the Operaaor will not perform any actions on the
+      - description: Disable TCMU can be set to true to disable the TCMU storage
+          driver.  This is recommended when there are multiple storage systems
```

```suggestion
          driver.  This is necessary/required when there are multiple storage systems
```

I'd like to see more absolute language here as using multiple TCMU systems at once will end badly.

---

good call - done